src/components/Calculator.js

```diff
@@ -1,62 +1,62 @@
-/* eslint-disable react/prefer-stateless-function */
-/* eslint-disable no-useless-constructor */
+/* eslint-disable import/extensions */
 
 import './Calculator.css';
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
-import Result from './result';
-import Button from './button';
-
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { output: '0' };
-    this.obj = {};
-  }
-
-  handleButtonClick = (buttonName) => {
-    this.obj = calculate(this.obj, buttonName);
-
-    if (this.obj.operation && !this.obj.next) {
-      this.setState({ output: this.obj.operation });
-    } else if (this.obj.next) {
-      this.setState({ output: this.obj.next });
-    } else if (this.obj.total) {
-      this.setState({ output: this.obj.total });
-    } else {
-      this.setState({ output: 0 });
+import Result from './Result';
+import Button from './Button';
+
+let obj = {};
+
+const Calculator = () => {
+  const [output, setOutput] = useState(0);
+
+  const handleButtonClick = (buttonName) => {
+    obj = calculate(obj, buttonName);
+
+    let outStr = '';
+    if (obj.total) {
+      outStr += obj.total;
+    }
+    if (obj.operation) {
+      outStr += obj.operation;
     }
```

Comment:
- [ ] Your project is not running locally, please fix the issue.
- [ ] Kindly fix the eslint errors without disabling them.

![error-1](https://user-images.githubusercontent.com/55635977/143405643-30120643-b243-4587-9343-c1d0465993f7.png)

![error-2](https://user-images.githubusercontent.com/55635977/143405650-ff53d1f7-8246-42d3-9f44-683218970fa3.png)